jobs/build/merge_ocp/Jenkinsfile

```diff
@@ -0,0 +1,126 @@
+
+// Expose properties for a parameterized build
+properties(
+    [
+        buildDiscarder(
+            logRotator(
+                artifactDaysToKeepStr: '',
+                artifactNumToKeepStr: '',
+                daysToKeepStr: '',
+                numToKeepStr: '1000')),
+        [
+            $class: 'ParametersDefinitionProperty',
+            parameterDefinitions: [
+                [
+                    name: 'VERSIONS',
+                    description: 'Version string for build without leading "v" (i.e. 4.0.0)',
+                    $class: 'hudson.model.StringParameterDefinition',
+                    defaultValue: "3.9,3.10,3.11,master"
```

Comment:
Just curious why `master` is listed, but `4.0.0` is in the example. Is this supposed to to be multiple choice, or is the `defaultValue` just there to give you something to erase as an example?

Reply:
Oh, `MERGE_VERSIONS` looks like `VERSIONS` is a comma separated input. So, maybe mention that in the `description` if that's correct?

Reply:
Oops. Copy error from the job I based this on. 
Fixed and removed master from the list in lieu of checking if a given version is the current master and then handling it specially.